packages/hint-compat-api/tests/css.ts

```diff
@@ -86,7 +86,7 @@ testHint(hintPath,
             reports: [
                 {
                     message: `'display: grid' is not supported by Edge < 16. Add 'display: -ms-grid' to support Edge 12+.`,
-                    position: { match: 'display: grid; /* Report */' }
+                    position: { match: 'grid; /* Report */' }
```

Comment:
I think we should be able to update the `match` logic to allow validating the new ranges as well. 

We can keep the current `string` value for matching the context, then add an optional `range` item to specify the sub-section we want to include. When `range` is present we'll match `endColumn` and `endLine` in addition to `column` and `line`. The necessary updates would be in `findPosition` of `@hint/utils-tests-helpers/src/hint-runner.ts`. 

For example here we could use:
```ts
{ match: 'display: grid; /* Report */', range: 'grid' }
```

What do you think?

Reply:
Sure, I like the idea